rust/frontend/src/server.rs

```diff
@@ -254,25 +271,50 @@ impl FrontendServer {
 // These handlers simply proxy the call and the relevant inputs into
 // the appropriate method on the `FrontendServer` struct.
 
+/// Health check endpoint that returns 200 if the server and executor are ready
+#[utoipa::path(
+    get,
+    path = "/api/v2/healthcheck",
+    responses(
+        (status = 200, description = "Success", body = String, content_type = "application/json"),
+        (status = 503, description = "Service Unavailable", body = ErrorResponse),
+    )
+)]
 async fn healthcheck(State(server): State<FrontendServer>) -> impl IntoResponse {
     server.metrics.healthcheck.add(1, &[]);
     let res = server.frontend.healthcheck().await;
     let code = match res.get_status_code() {
         tonic::Code::Ok => StatusCode::OK,
         _ => StatusCode::SERVICE_UNAVAILABLE,
     };
-
     (code, Json(res))
 }
 
+/// Heartbeat endpoint that returns a nanosecond timestamp of the current time. Useful for making sure the client remains connected.
```

Comment:
This is not intended to be "Useful for making sure the client remains connected". It just returns the TS, we don't really encourage people to use it this way for keep-alive style reasons. We just have this for debug moreso.

Reply:
Ah, I copied that verbatim out of the docs - so we should update there: 

https://docs.trychroma.com/docs/run-chroma/persistent-client